tools/tasks.py

```diff
@@ -1065,7 +1065,7 @@ def disarm(self):
         self.p = None
 
 
-def do_upload_and_exit(path, url, proxy):
+def do_upload_and_exit(path, url, proxy, keepZip):
```

Comment:
keepZip is unused here

Reply:
https://github.com/couchbase/sync_gateway/pull/6119/commits/4bb03785f8f89dda3648cfe2c45fca796c80ff2f